### PR TITLE
Accuracy tests for rewards

### PR DESCRIPTION
**Pull request by AlexChetverov**

### CODE REVIEW

File: test/parallel/xyk-pallet.API.poolLiqudity.multipleUserRewards.test.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+    testUser1.keyRingPair.address,
+    liqIdPromPool.toString()
+  );
+
+  const rewardsAmountUser2 = await mangata.calculateRewardsAmount(
+    testUser2.keyRingPair.address,
+    liqIdPromPool.toString()
+  );
+
+  expect(rewardsAmountUser1.mul(new BN(2))).bnLte(rewardsAmountUser2);
```
Comment (goncer) [inline test/parallel/xyk-pallet.API.poolLiqudity.multipleUserRewards.test.ts:128]: Plz expect that is > than zero. 
Why lte? perhaps we can be a bit more accurate? like the diff is less than 1000 ?

File: test/parallel/xyk-pallet.API.poolLiqudity.sepClaimRewards.test.ts [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    liqIdPromPool
+  );
+  const rewardsClaimedUser1 = rewardsInfoUser1After.rewardsAlreadyClaimed.sub(
+    rewardsInfoUser1Before.rewardsAlreadyClaimed
+  );
+  const rewardsClaimedUser2 = rewardsInfoUser2After.rewardsAlreadyClaimed.sub(
+    rewardsInfoUser2Before.rewardsAlreadyClaimed
+  );
+
+  expect(rewardsClaimedUser1).bnEqual(rewardsClaimedUser2);
```
Comment (goncer) [inline test/parallel/xyk-pallet.API.poolLiqudity.sepClaimRewards.test.ts:143]: can you plz check if the storage are almost identical? I think they should be.

**Review by goncer — COMMENTED** [review]
Looks awesome, just a few minor checks!

File: test/parallel/xyk-pallet.API.poolLiqudity.sepClaimRewards.test.ts (hunk 2 above)
Comment (AlexChetverov) [reply test/parallel/xyk-pallet.API.poolLiqudity.sepClaimRewards.test.ts:143]: Done

**Review by goncer — APPROVED** [review]

**Pull request merged**